File: services/webhook/webhook_setting.js
const superagent = require('superagent')

superagent
  .put('https://mailsac.com/api/private-address-forwarding/bclinton@mailsac.com')
  .set('Mailsac-Key', 'YOUR_API_KEY_HERE')
  .send({
      webhook: "https://mywebsite.com/webhooks"
  })
  .then((res) => {
      console.log(res.body)
  })
  .catch(err => {
      console.log(err.message)
  })

  /*
  { _id: 'bclinton@mailsac.com',
  owner: 'mailsac_user',
  forward: null,
  enablews: false,
  webhook: 'https://mywebsite.com/webhooks',
  webhookSlack: null,
  webhookSlackToFrom: null,
  catchAll: null,
  password: null,
  info: '',
  created: '2020-08-19T23:31:15.664Z',
  updated: '2020-08-25T14:04:34.261Z' }
*/

File: services/webhook/webhook_setting.py
import requests
from pprint import pprint

headers = {'Mailsac-Key': 'YOUR_API_KEY_HERE'}
url = 'https://mailsac.com/api/private-address-forwarding/bclinton@mailsac.com'
data = { 'webhook':'https://mywebsite.com/webhooks' }

r = requests.put(url, data=data, headers=headers)
pprint(r.json())
'''
{'_id': 'bclinton@mailsac.com',
 'catchAll': None,
 'created': '2020-08-19T23:31:15.664Z',
 'enablews': False,
 'forward': None,
 'info': '',
 'owner': 'mailsac_user',
 'password': None,
 'updated': '2020-08-25T14:11:14.695Z',
 'webhook': 'https://mywebsite.com/webhooks',
 'webhookSlack': None,
 'webhookSlackToFrom': None}
 '''
